<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonGetLacticAcidFlex">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Query Lactic Acid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetPhFlex">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Query pH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCheckFlex2Conn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Check FLEX2 Conn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>pH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPh">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lactic Acid Conc</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLacticAcidConc">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFlex2ConnStatus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Flex2 Conn Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCheckQfConn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Check FLEX2 Conn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>QF Conn Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQfConnStatus">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Connected</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQfIpAddr">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10.183.169.70</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>QF IP Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFlex2IpAddr">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>FLEX2 IP Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQueryQFflow">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Query Flow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnToSelQf">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Conn to Sel QF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFlow">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
